### PR TITLE
FF-RN-24Jan23

### PR DESCRIPTION
**Pull request by Clare-Wilkinson**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by deepakbolangady — APPROVED** [review]
LGTM! I noticed what looks like an empty section. Please take a look. Otherwise, approved.

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -13,6 +13,24 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+This release does not include early access features.
+
+### Fixed issues
+- The Android SDK has been updated to version 1.0.18. This fixes a bug that caused unhandled exception errors due to duplicate callbacks during the SDK initialization. (FFM-6395)
+- The Flutter SDK has been updated to version 1.0.8. This includes the following:
+  - Fixed a bug that caused applications to shut down in response to API errors caused by no internet connection. (FFM-6395)
+  - Fixed a bug that caused streaming to stop working if internet connectivity was lost. (FFM-6395) 
+
+#### Feature Flag SDKs
+
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:33]:
```suggestion

This release does not include fixes for Feature Flag SDKs
```

**Review by douglas-j-bothwell — APPROVED** [review]
One sug, that 'Feature Flag SDKs' heading looks a little lonely without any content. Otherwise approved.

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -13,6 +13,24 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+This release does not include early access features.
+
+### Fixed issues
+- The Android SDK has been updated to version 1.0.18. This fixes a bug that caused unhandled exception errors due to duplicate callbacks during the SDK initialization. (FFM-6395)
+- The Flutter SDK has been updated to version 1.0.8. This includes the following:
+  - Fixed a bug that caused applications to shut down in response to API errors caused by no internet connection. (FFM-6395)
+  - Fixed a bug that caused streaming to stop working if internet connectivity was lost. (FFM-6395) 
+
+#### Feature Flag SDKs
```
[the remark this answers is not in the document]
Comment (Clare-Wilkinson) [reply release-notes/feature-flags.md]: Thank you! It was meant to go above the content. Fixed now :)

File: release-notes/feature-flags.md (hunk 1 above)
Comment (Clare-Wilkinson) [reply release-notes/feature-flags.md]: Thank you! It was meant to go above the content. Fixed now :)

**Comment by Clare-Wilkinson** [comment]
Fixed heading as per review, now merging.

**Pull request merged**